docs/src/components/Palette.tsx

```diff
@@ -0,0 +1,282 @@
+import React from "react";
+import styled, { css } from "styled-components";
+import convertHexToRgba from "@kiwicom/orbit-design-tokens/lib/convertHexToRgba.js";
+import useMediaQuery from "@kiwicom/orbit-components/lib/hooks/useMediaQuery";
+import { defaultTokens } from "@kiwicom/orbit-design-tokens";
+import { Grid, Stack, Text } from "@kiwicom/orbit-components";
+import { CopyToClipboard } from "react-copy-to-clipboard";
+
+import CopyIcon from "../images/icons/CopyIcon.svg";
+import { copyTimeout } from "../utils/common";
+import Switch from "./Switch";
+
+const isLight = (hex: string) => {
+  const { red, green, blue } = convertHexToRgba(hex, 100).match(
+    /\((?<red>\d+), (?<green>\d+), (?<blue>\d+)/,
+  ).groups;
+
+  // HSP equation from http://alienryderflex.com/hsp.html
+  const hsp = Math.sqrt(0.299 * (red * red) + 0.587 * (green * green) + 0.114 * (blue * blue));
```

Comment:
Definitely, but feel free to use this a bit of custom code. I'm not sure if `color2k` supports the same use case @CollierCZ wants to cover. Anyway, it can be updated next week when I'll integrate the design tokens to the docs.

Reply:
I'll leave it for now then.